README.Rmd

```diff
@@ -15,75 +17,67 @@ knitr::opts_chunk$set(
 )
 ```
 
-# Rewrite of [r-polars](https://github.com/pola-rs/r-polars)
+# polars
 
-## Motivation
+<!-- TODO: add link to discord -->
 
-I have been developing r-polars for over a year, and I felt that a significant rewrite was necessary.
-r-polars is a clone of [py-polars](https://github.com/pola-rs/polars/tree/main/py-polars) /
-[polars-python](https://github.com/pola-rs/polars/tree/main/crates/polars-python),
-but the package structure is currently quite different.
-Therefore, it was difficult to keep up with frequent updates.
+<!-- badges: start -->
+[![R-multiverse status](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fcommunity.r-multiverse.org%2Fapi%2Fpackages%2Fpolars&query=%24.Version&label=r-multiverse)](https://community.r-multiverse.org/polars)
+[![R-universe status badge](https://rpolars.r-universe.dev/badges/polars)](https://rpolars.r-universe.dev)
+[![CRAN status](https://www.r-pkg.org/badges/version/polars)](https://CRAN.R-project.org/package=polars)
+[![Docs dev version](https://img.shields.io/badge/docs-dev-blue.svg)](https://pola-rs.github.io/r-polars)
+<!-- badges: end -->
 
-I thought that now, around the release of Python Polars 1.0.0, is a good time for a complete rewrite, so I decided to try it.
+## Overview
 
-There are several reasons to rewrite r-polars on both the Rust and R sides.
+Polars is a DataFrame interface on top of an OLAP Query Engine implemented in Rust using
+[Apache Arrow Columnar Format](https://arrow.apache.org/docs/format/Columnar.html) as the memory
+model.
 
-### Rust side
+- Lazy | eager execution
+- Multi-threaded
+- SIMD
+- Query optimization
+- Powerful expression API
+- Hybrid Streaming (larger-than-RAM datasets)
+- Rust | Python | NodeJS | R | ...
```

Comment:
Similarly, I expect most average R users to not be knowledgeable about multithreading, SIMD, query optimization, etc.